packages/select/src/components/multi-select/multiSelect2.tsx

```diff
@@ -167,6 +167,7 @@ export class MultiSelect2<T> extends AbstractPureComponent2<MultiSelect2Props<T>
         const inputProps = {
             ...tagInputProps.inputProps,
             className: classNames(tagInputProps.inputProps?.className, Classes.MULTISELECT_TAG_INPUT_INPUT),
+            role: "combobox",
```

Comment:
actually, I think these attributes need to go on the `<div>` wrapping the TagInput on L203. That's where we set `aria-haspopup`, so it would make sense to add the other aria attributes on the same element.

Related to this, note the recent PR that merged https://github.com/palantir/blueprint/pull/5300. You should merge `develop` to get that change, and then explicitly set `<Popover2 popupKind="listbox">` below